client/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google</title>
+    <link
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+			rel="stylesheet"
+			integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
+			crossorigin="anonymous"
+		/>
+    <link rel="stylesheet" href="assets/style.css" />
+    <link rel="icon" href="assets/favicon.png" />
+
+    <script src="./assets/script.js"></script>
+    <script defer="" src="./assets/index.js"></script>
```

Comment:
defer should be true or false. If the attribute isn't there, it defaults to false. If Attribute is there with no arguments, it defaults to true.

<script defer src="./assets/index.js"></script>